workspace/utilities/lib/struct-modal.xsl

```diff
@@ -0,0 +1,131 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xsl:stylesheet version="1.0" xmlns:xsl="http://www.w3.org/1999/XSL/Transform">
+
+<!-- 
+	Creates a modal, either for a page or a simple element.
+
+	Content
+		modal: The actual content of the modal
+		close: The element that will trigger a closing action
+
+	Options
+		id: Unique identifier used to easily target a specific modal
+		is-visible: Determines if the modal starts visible or not
+		is-page: Changes the behavior of the modal to make it work as a page
+
+	Dependencies
+		modules/change-state.js
+		modules/auto-change-state-click.js
```

Comment:
So we drop modules/auto-modal.js modules/popup.js and transition/popup-transition.js ?

Reply:
modules/auto-modal.js **Most likely,** except if someones see something special there
modules/popup.js **No,** but would need a PR with updated version and renaming
transition/popup-transition.js **No,** same as above

I'll update

Reply:
@fhamon drop means "not a deps" or "delete from repo because not needed anymore?"

@dominic-blain please link to relevant prs.

Reply:
@nitriques https://github.com/DeuxHuitHuit/framework.js-modules/pull/239